source/policies/security.rst

```diff
@@ -8,4 +8,45 @@ permalink: /policies/security.html
 Security Policy
 ===============
 
-Coming soon...
+This document explains how Doctrine security issues are handled by the core team and how
+you should report a security issue if you think you have discovered one.
+
+Reporting a Security Issue
+--------------------------
+
+If you think that you have found a security issue in Doctrine, please don't use the
+issue tracker in GitHub and don't publish it publicly. Instead, all security issues
+must be sent to `security@doctrine-project.org <mailto:security@doctrine-project.org>`_.
```

Comment:
I'd probably still add a GPG public key that we can use for this - think it is needed?

Reply:
Sure, which one do you want to use? I am not sure I trust myself enough yet to be trusted with that :)

Reply:
I am going to merge this as it is now and we can follow up with an enhancement once we decide what to do here.